_includes/head.html

```diff
@@ -6,14 +6,14 @@
     <meta name="description" content="">
     <meta name="author" content="">
 
-    <title>Open Source Design</title>
+    <title>Open Source Design events</title>
```

Comment:
<insert opportunity to lobby for bringing all of the website into one repository/>